src/anbox/cli.h

```diff
@@ -243,6 +243,9 @@ class Command : public DoNotCopyOrMove {
   /// @brief description returns a longer string explaining the command.
   virtual Description description() const;
 
+  /// @brief hidden returns if the command is hidden from the user or not.
+  virtual bool hidden() const;
```

Comment:
I would rather prefer virtual Visibility visibility() const; with enum class Visibility { visible, hidden };. With that, ctor invocations are easier to read, e.g.: Command("test", "test", "test", Visibility::hidden); We could also extend to cases like Visibility::visible_if_root.